python/pyarrow/includes/libarrow.pxd

```diff
@@ -44,6 +44,11 @@ cdef extern from "arrow/util/key_value_metadata.h" namespace "arrow" nogil:
         c_bool Contains(const c_string& key) const
 
 
+cdef extern from "arrow/util/decimal.h" namespace "arrow" nogil:
+    cdef cppclass CDecimal128" arrow::Decimal128":
+        c_string ToString(int32_t scale) const
```

Comment:
Hmm, at some point we should be able to pretty-print scalars in C++.

Reply:
Created jira https://issues.apache.org/jira/browse/ARROW-9279